packages/odie-client/src/hooks/use-get-combined-chat.ts

```diff
@@ -39,18 +39,28 @@ export const useGetCombinedChat = ( canConnectToZendesk: boolean ) => {
 	);
 
 	const [ mainChatState, setMainChatState ] = useState< Chat >( emptyChat );
+	const [ isEnabled, setIsEnabled ] = useState( true );
+	const chatStatus = mainChatState?.status;
 	const getZendeskConversation = useGetZendeskConversation();
-	const { data: odieChat, isLoading: isOdieChatLoading } = useOdieChat( Number( odieId ) );
+	const { data: odieChat, isFetching: isOdieChatLoading } = useOdieChat( Number( odieId ) );
 	const { startNewInteraction } = useManageSupportInteraction();
 	const { trackEvent } = useOdieAssistantContext();
 
 	useEffect( () => {
+		if ( isOdieChatLoading || ! isEnabled ) {
+			return;
+		}
+		if ( chatStatus === 'loaded' ) {
+			setIsEnabled( false );
```

Comment:
Do we need this state? Wouldn't returning do the job?

Reply:
Added this state to limit the amount of runs of this hook.
We want it to run once per chat. But the chat status can change from loaded to loading and then back to loaded when sending messages like it is [here](https://github.com/Automattic/wp-calypso/blob/05bd96f3f3725846bffbef954efbcc66f2aec587/packages/odie-client/src/hooks/use-send-zendesk-message.ts#L26).
This was leading to unpredictable behaviour and the purpose of this PR is to limit that.

Reply:
Can you add a comment to the effect of "status can go back to `loading` after being `loaded`". Because now it seems the state is extraneous, and I'd remove it if I came across it.